layouts/stripe-checkout.html

```diff
@@ -0,0 +1,146 @@
+{{ define "main" }}
+  <div class="article-content mx-auto mb-8 max-w-2xl px-4 py-12">
+    <h1 class="mb-4 font-serif text-3xl font-bold sm:text-4xl">
+      {{ .Title }}
+    </h1>
+
+    <div class="mb-3 flex items-baseline gap-2">
+      <div class="flex items-center gap-2">
+        <span class="text-3xl font-bold text-g-9 sm:text-4xl">
+          {{ .Params.price }}
+        </span>
+
+        <span
+          class="rounded bg-g-2 px-2 py-0.5 text-xs uppercase tracking-wider text-g-7"
+          >{{ .Params.period }}</span
+        >
+      </div>
+    </div>
+
+    <div class="mb-6 space-y-2 text-base text-g-9">
+      <p>Newsletters sent weekly, except for holidays.</p>
+      <ul class="list-disc pl-5">
+        <li>Email used for purchase will be where newsletters are sent.</li>
+        <li>All purchases are final and non-refundable.</li>
+        <li>This is not a tax-deductible donation.</li>
+      </ul>
+      <p>
+        Interested in a yearly group subscription? Want to change your
+        newsletter email? Have other questions? Email
+        <a href="mailto:membership@spotlightpa.org"
+          >membership@spotlightpa.org</a
+        >.
+      </p>
+    </div>
+
+    <form id="payment-form">
+      <div id="payment-element"></div>
+
+      <button id="submit" class="btn-signup mt-6 w-full">
+        <span id="button-text">Subscribe</span>
+        <span id="spinner" class="hidden">Processing...</span>
+      </button>
+      <p
+        class="mt-2 flex items-center justify-center gap-1 text-center text-xs text-g-7"
+      >
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke-width="1.5"
+          stroke="currentColor"
+          class="h-4 w-4"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            d="M16.5 10.5V7.5a4.5 4.5 0 10-9 0v3h9zM6.75 10.5h10.5a1.5 1.5 0 011.5 1.5v6.75a1.5 1.5 0 01-1.5 1.5H6.75a1.5 1.5 0 01-1.5-1.5V12a1.5 1.5 0 011.5-1.5z"
+          />
```

Comment:
Can we use the Font Awesome lock icon here instead by adding `@fontawesome/solid/lock.svg` to svgs.html?